### ISSUE TITLE
Runtime Model / Code: AccessDenied

### ISSUE THREAD

**Issue by ibnHatab** [tokio-rs/doc-blitz#93]
https://tokio.rs/docs/getting-started/runtime-model/
response with Code: AccessDenied

According to wayback archive last visible version is 18 Now 2018
https://web.archive.org/web/20181027230142/https://tokio.rs/docs/getting-started/runtime-model/

--------------------------------------------------
403 Forbidden

    Code: AccessDenied
    Message: Access Denied
    RequestId: A498E228BD53C137
    HostId: 5Ypr/gVNRHJx2WFBVqCdUkP/ARdBqPLMhBch6iBrdirU1G2VkBgTPmwIEog6W/c4/2QQZTqXifA=

An Error Occurred While Attempting to Retrieve a Custom Error Document

    Code: AccessDenied
    Message: Access Denied

**Comment by rylev** [comment]
The URL no longer exists. It should return a 404, but currently does not - that issues is tracked [here](https://github.com/tokio-rs/website/issues/212).

Where did you see this url? Did you have it bookmarked or was it linked somewhere?

**Comment by carllerche** [comment]
Yes, I know that S3 can do redirects. I wonder if it would be possible to find all old links that are being linked to.
